src/System.Windows.Forms/src/System/Windows/Forms/ComboBox.cs

```diff
@@ -4687,25 +4690,33 @@ internal class ComboBoxItemAccessibleObject : AccessibleObject {
         internal class ComboBoxItemAccessibleObjectCollection : Hashtable {
 
             private ComboBox _owningComboBoxBox;
+            private ObjectIDGenerator _idGenerator = new ObjectIDGenerator();
```

Comment:
```suggestion
            private readonly ObjectIDGenerator _idGenerator = new ObjectIDGenerator();
```
Arguably it can be made static too

Reply:
If you make it static you need a cleanup strategy to prevent unbounded memory growth (aka memory leak)

Reply:
I had to look up the source for it, you're right.